BioCro/DARPA/plot_MA.R

```diff
@@ -0,0 +1,87 @@
+##  Function to make figures
+
+plot_MA <- function(settings){
+  
+  library(ggplot2)
+  library(tidyr)
+  library(dplyr)
+
+  #load prior.distns, jagged.data, trait.mcmc (new), and trt.match
+  load(file.path(settings$pfts$pft$outdir, "prior.distns.Rdata"))
+  load(file.path(settings$pfts$pft$outdir, "jagged.data.Rdata"))
+  load(file.path(settings$pfts$pft$outdir, "trait.mcmc.Rdata"))
+  load(file.path(settings$pfts$pft$outdir, "trt.match.Rdata"))
+  
+  #turn all into lists of length n objects so plot_prior_posterior() will run evenly
+  traits <- names(trt.match) # not all jagged data was processed
```

Comment:
~~What does this comment about jagged data mean?~~

Ah, I see now. Because not all of the variables in the priors goes through the meta-analysis because there's no data. Do we still want to plot the prior distributions for those extra variables? I'm not sure if that would be useful at all.

Reply:
You are correct that they are not as useful when primarily presenting data summaries. But the priors are useful - even in the absence of data - when they show how the model was parameterized.